README.md

```diff
@@ -58,5 +42,120 @@ You can also track upcoming changes using the [awaiting-deployment](https://gith
+| Python <br/> Ruby | 5 most popular `major.minor` versions |
+| PyPy      | 3 most popular `major.minor` versions |
+| .NET Core | 2 latest LTS versions and 1 latest version. For each feature version only latest patch is installed |
+| GCC <br/> GNU Fortran <br/> Clang <br/> GNU C++ | 3 latest major versions |
+| Android NDK | 1 latest non-LTS, 2 latest LTS versions |
+| Xcode     | - all OS compatible versions side-by-side <br/> - for beta, GM versions - latest beta only <br/> - old patch versions are deprecated in 3 months |
+
+### Image Deprecation Policy
+- Images begin the deprecation process of the oldest image label once a new GA OS version has been released.
+- Deprecation process begins with an announcement that sets a date for deprecation
+- As it gets closer to the date, GitHub begins doing scheduled brownouts of the image
+- DUring this time there will be an Announcement pinned in the repo to remind users of the deprecation.
+- Finally GitHub will deprecate the image and it will no longer be available
+
+### Preinstallation Policy
+In general, these are the guidelines we follow when deciding what to pre-install on our images:
+
+- Popularity: widely-used tools and ecosystems will be given priority.
+- Latest Technology: recent versions of tools will be given priority.
+- Deprecation: end-of-life tools and versions will not be added.
+- Licensing: MIT, Apache, or GNU licenses are allowed.
+- Time & Space on the Image: we will evaluate how much time is saved and how much space is used by having the tool pre-installed.
+- Support: If a tool requires the support of more than one version, we will consider the cost of this maintenance.
+
+### Default Version Update Policy
+- In general, once a new version is installed on the image, we announce the default version update 2 weeks prior to deploying it.
+- For potentially dangerous updates, we may extend the timeline up to 1 month between the announcement and deployment.
+
+## How to Interact with the Repo
+- **Issues**: To file a bug report, or request tools to be added/updated, please [open an issue using the appropriate template](https://github.com/actions/virtual-environments/issues/new/choose)
+- **Discussions**: If you want to share your thoughts about image configuration, installed software, or bring a new idea, please create a new topic in a [discussion](https://github.com/actions/virtual-environments/discussions) for a corresponding category. Before making a new discussion please make sure no similar topics were created earlier.
+- For general questions about using the runner images or writing your Actions workflow, please open requests in the [GitHub Actions Community Forum](https://github.community/c/github-actions/41).
+
+## FAQs
+<details>
+   <summary><b><i>What images are available for GitHub Actions and Azure DevOps?</b></i></summary>
+
+The availability of images for GitHub Actions and Azure DevOps is different. See documentation for more details:

```

Comment:
I would say the opposite — they are the same but there could be differences due to the discrepancy in deprecation periods.

Reply:
ok will update wording here